src/main/java/com/blackducksoftware/integration/jira/common/settings/model/PluginGroupsConfigModel.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Black Duck JIRA Plugin
+ *
+ * Copyright (C) 2019 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.jira.common.settings.model;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.util.Stringable;
+
+public class PluginGroupsConfigModel extends Stringable {
+    public static final String BLACK_DUCK_GROUPS_LIST_DELIMETER = ",";
```

Comment:
Is it possible for a BD group to contain a comma? If so, we could have bugs regarding this in the future.